server/utils/nunjucksSetup.ts

```diff
@@ -5,6 +5,9 @@ import * as pathModule from 'path'
 import { PrisonApiOffenderSentenceAndOffences } from '../@types/prisonApi/prisonClientTypes'
 import config from '../config'
 
+// TODO the use of nunjucks-date-filter is raising a deprecation warning, some dates are in this format 12/12/2030 ->
```

Comment:
@yasinmustafa  do we need to add a ticket for this one

Reply:
yes - some tech debt